iTwin/iTwinUI
# fix(Toast): Added checking for `prefers-reduced-motion`

## Changes

Removes transitions for `Toast` component when `prefer-reduced-motion` is active.
Closes #927 

## Testing

1. Open up the `Toast` story in storybook
2. Right click & Inspect page
3. Click the 3 dot menu in the top right of the inspect panel & select More tools > Rendering
4. In the Rendering tab, find Emulate CSS Media feature prefers-reduced-motion
5. See the changes made by opening and closing different toasts

![ToastReduceMotionDemo](https://user-images.githubusercontent.com/78564221/213307467-9789fb56-f6aa-4790-964f-86ae1be11210.gif)

## Docs

Changeset added
